lib/helpers.py

```diff
@@ -556,13 +571,21 @@ def read_csv(self, audience, source, date, chunk_filter_fn=None):
             # remove events without a user id
             filtered_chunk = filtered_chunk[filtered_chunk['user_id'].str.len() == USER_ID_LENGTH]
 
+            if self.export_user_ids:
+                test_users_chunk = filtered_chunk[filtered_chunk['ab_test_group'] == TEST][['user_id']].drop_duplicates()
+                control_users_chunk = filtered_chunk[filtered_chunk['ab_test_group'] == CONTROL][['user_id']].drop_duplicates()
+                self.test_users = pd.concat([self.test_users, test_users_chunk], ignore_index=True, verify_integrity=True)
```

Comment:
Please, make these just variables and `_save_exported_ids` arguments: there's no need to make them a class property, since they are used exactly within one method (`read_csv`) via exactly one call (to "private" `_save_exported_ids`)

Reply:
:+1: